packages/react-dom/src/events/SyntheticEvent.js

```diff
@@ -170,12 +170,12 @@ export const MouseEventInterface = {
   button: 0,
   buttons: 0,
   relatedTarget: function(event) {
-    return (
-      event.relatedTarget ||
-      (event.fromElement === event.srcElement
+    if (!event.relatedTarget && event.fromElement)
```

Comment:
What if `fromElement` is `null` but `toElement` is not? Could that happen?

I wonder if checking something like `event.relatedTarget !== undefined` makes more sense.

Reply:
Good point - replaced with `(event.relatedTarget === undefined)` in ccdf9c2f249ff017ce9633fec430c8cd6a68d386